packages/core/src/EntityManager.ts

```diff
@@ -163,7 +164,8 @@ export class EntityManager<D extends IDatabaseDriver = IDatabaseDriver> {
   async find<
     Entity extends object,
     Hint extends string = never,
-  >(entityName: EntityName<Entity>, where: FilterQuery<Entity>, options: FindOptions<Entity, Hint> = {}): Promise<Loaded<Entity, Hint>[]> {
+  >(entityName: EntityName<Entity>, where: FilterQuery<Entity>, _options: FindOptions<Entity, Hint> = {}): Promise<Loaded<Entity, Hint>[]> {
+    let options = this.mergeOptions<FindOptions<Entity, Hint>>(_options);
```

Comment:
i also dont think we need the `mergeOptions` method, its already a one-liner (and if we do, it cant be a public method). i would say it should be just this instead:

```suggestion
    options.schema ??= this._schema;
```

Reply:
I agree that it doesn't need to be public, but I added the method to allow for other similar options to be applied easily. But that can be done when/if it's needed. I will implement your feedback.